builder/qemu/builder.go

```diff
@@ -154,8 +154,16 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 		if runtime.GOOS == "windows" {
 			b.config.Accelerator = "tcg"
 		} else {
-			b.config.Accelerator = "kvm"
+			if fp, err := os.Open("/dev/kvm"); err != nil {
```

Comment:
I see, thanks for the explanation. The info surrounding `open` is really interesting. This would be a good thing to add in a comment.

I did a bit more digging and it appears that `tcg` is the default accelerator in ubuntu 16 qemu (possibly others).

Would you also be able to update the documentation, since it is wrong now?

> By default "kvm" is used.

This should be changed to something like:

> If no accelerator is specified packer will try to use `kvm` on linux, but will default to `tcg` when it is not available.

I'll add this and merge momentarily.